solace-message-router/azuredeploy.json

```diff
@@ -513,14 +596,14 @@
         "networkProfile": {
           "networkInterfaces": [
             {
-              "id": "[resourceId('Microsoft.Network/networkInterfaces', concat(variables('nicName'), copyindex()))]"
+              "id": "[resourceId('Microsoft.Network/networkInterfaces', concat(parameters('vmAndLoadbalancerExposure'),variables('nicName'), copyindex()))]"
             }
           ]
         }
       }
     },
     {
-      "apiVersion": "2019-12-01",
+      "apiVersion": "2019-07-01",
```

Comment:
why are we reverting to an older version?

Reply:
The best practice check script didn't recognize `"apiVersion": "2019-12-01"` so changed it to the closest it listed. Sounds like not an issue so reverting.